src/bosh-softlayer-cpi/softlayer/pool/softlayer_pool_creator.go

```diff
@@ -107,46 +110,49 @@ func (c *softLayerPoolCreator) createFromVMPool(agentID string, stemcell bslcste
 			if err != nil {
```

Comment:
The first one is checking for creating VM by SoftLayer, and the second one is checking the created VM to register to VPS server. So two interactives with others system need to check error separately.